cmd/admin-heal-ops.go

```diff
@@ -577,7 +577,8 @@ func (h *healSequence) queueHealTask(path string, healType madmin.HealItemType)
 func (h *healSequence) healItemsFromSourceCh() error {
 	h.lastHealActivity = UTCNow()
 
-	if err := h.healItems(); err != nil {
+	bucketsOnly := true // heal buckets only, not objects.
```

Comment:
So for background healing, we will heal only the buckets and not objects? Is this a new change?

Reply:
No it is not @Praveenrajmani background healing still runs on its own. This is to ensure that disks are healed right after object layer is initialized for reads.

In our latest release we don't heal disks automatically for 10mins, even then it won't work correctly because the healState is not initialized yet. All of these changes were done in the locking PR change, but this PR is needed to ensure that we don't heal all objects suddenly when server starts. Give it some time and let the background heal do the work. Rest of the bucket and disk healing can be done automatically without this wait. 

That is why we have test cases in verify-healing